package.json

```diff
@@ -5,13 +5,19 @@
   "main": "index.js",
   "scripts": {
     "test": "lerna run test",
+    "test-ci": "npm-run-all test coverage:*",
+    "coverage:create-file": "echo '' > ./coverage_report/coverage_merged.info",
+    "coverage:merge-reports": "lcov-result-merger './coverage_report/*/lcov.info' './coverage_report/coverage_merged.info'",
+    "coverage:upload-coveralls": "cat ./coverage_report/coverage_merged.info | coveralls",
     "cleanInstall": "lerna exec npm install --ignore-scripts",
     "bootstrap": "lerna bootstrap",
     "start": "lerna run start  --parallel --stream --scope=@raincatcher/demo-*",
     "docs": "lerna exec -- typedoc --out docs/ --excludePrivate",
     "lint": "tslint '*/*/src/**/*.ts' '*/*/test/**/*.ts'",
     "build": "lerna run build",
-    "clean": "lerna run clean"
+    "clean": "npm-run-all clean:*",
```

Comment:
With amount of new tasks here we may probably need to go to grunt or some scripts. 
This can easily become unmanageable. 

@paolobueno WDYT?